org.eclipse.scanning.test/src/org/eclipse/scanning/test/scan/servlet/AbstractServletTest.java

```diff
@@ -174,15 +174,16 @@ public void disconnect()  throws Exception {
 			servlet.disconnect();
 		}
 	}
-	
+
+	@SuppressWarnings("sonar:S1192")
 	protected ScanBean createStepScan() throws IOException {

```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'java.io.IOException', as it cannot be thrown from method's body. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)